<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vogleditor_QProgramExplorer</class>
 <widget class="QWidget" name="vogleditor_QProgramExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="programListBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="shaderListBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="saveShaderButton">
       <property name="text">
        <string>save changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="shaderTextEdit">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
